MaxText/experimental/rl/README.md

```diff
@@ -1,13 +1,15 @@
 # Generalized Reinforcement Policy Optimization (GRPO) in MaxText
 
-This directory contains code and documentation for **GRPO**, a reinforcement learning algorithm designed to optimize language model policies within the MaxText framework. GRPO enables training language models to perform specific tasks by optimizing for a reward signal, going beyond standard language modeling objectives. This implementation leverages techniques such as **Fast Sharded Data Parallelism (FSDP)** for training and **Data Parallelism (DP) + Tensor Parallelism (TP)** for inference.
+This directory contains code and documentation for **GRPO**, a reinforcement learning algorithm designed to optimize language model policies within the MaxText framework. GRPO enables training language models to perform specific tasks by optimizing for a reward signal, going beyond standard language modeling objectives. This implementation leverages techniques such as **Fully Sharded Data Parallelism (FSDP)** for training and **Data Parallelism (DP) + Tensor Parallelism (TP)** for inference.
 
 ## Key Concepts
 
 *   **GRPO (Generalized Reinforcement Policy Optimization):** A policy optimization algorithm that uses reinforcement learning principles to improve the quality of a language model's responses based on a provided reward signal.
-*   **FSDP (Fast Sharded Data Parallelism):** A technique for distributing model parameters across multiple devices during training, allowing for efficient training of large models.
+*   **FSDP (Fully Sharded Data Parallelism):** A technique for distributing model parameters across multiple devices during training, allowing for efficient training of large models.
 *   **DP (Data Parallelism):** A technique for distributing data across multiple devices during inference, allowing for efficient inference of large models.
-*   **Pathways:** A system that allows to orchestrate computations across multiple TPU slices, enabling training and inference on very large TPU topologies.
+*   **Sharding:** For more information on sharding, refer to [getting_started/Sharding.md](https://github.com/AI-Hypercomputer/maxtext/blob/main/getting_started/Sharding.md)
+*   **[Pathways](https://cloud.google.com/ai-hypercomputer/docs/workloads/pathways-on-cloud/pathways-intro):** A system that allows to orchestrate computations across multiple TPU slices, enabling training and inference on very large TPU topologies.
+*   **inference_rollouts:** The frequency in training steps to reshard and copy training parameters to the inference mesh. Must be an integer between 1 and `steps`.
```

Comment:
This is different from previous concepts above in that its a config parameter. I suggest to start the sentence with that info. Also, what is steps ? is that STEPS ? is that the total number of training steps to execute ?